tests/test_embedders.py

```diff
@@ -1,22 +1,29 @@
 from unittest import TestCase
+from unittest.mock import patch
 
-from sister.word_embedders import FasttextEmbedding
+import numpy as np
 
 
 class FasttextEmbeddingCase(TestCase):
 
     def setUp(self):
-        self.embedding = FasttextEmbedding(lang="en")
+        embedding_patcher = patch('sister.word_embedders.FasttextEmbedding')
+        self.embedding = embedding_patcher.start()(lang='en')
+        self.embedding.get_word_vector.return_value = np.random.rand(300)
+        self.embedding.get_word_vectors.side_effect = lambda words: np.random.rand(len(words), 300)
```

Comment:
Why isn't this 
```py
self.embedding.get_word_vectors.return_value = np.random.rand(len(words), 300)
```
?

Reply:
When a test runner executes the setUp method, the words variable isn't available.
So if we try to set return_value, we might do this: `return_value = np.random.rand(3, 300)`.
But I'd like to avoid hard-coding. That's why I choose to do that above.

FYI: [unittest.mock Calling](https://docs.python.org/3/library/unittest.mock.html#calling)